src/components/organisms/Onboarding/StepActions.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FormEvent, ReactElement } from 'react'
+import Alert from '../../atoms/Alert'
+import Button from '../../atoms/Button'
+import Loader from '../../atoms/Loader'
+
+import styles from './StepActions.module.css'
+
+interface IStepAction {
+  buttonLabel: string
+  buttonAction: (e: FormEvent) => void
+  successMessage?: string
+  loading: boolean
+  completed: boolean
+  loadingMessage?: string
+}
+
+export default function StepActions({
+  buttonLabel,
+  buttonAction,
+  successMessage,
+  loading,
+  completed,
+  loadingMessage
+}: IStepAction): ReactElement {
+  return (
+    <div className={styles.container}>
+      {loading ? (
+        <Loader message={loadingMessage || 'Loading...'} />
```

Comment:
nitpick(non-blocking): We could also use a dedicated variable here that contains the `'Loading...'` text. It's cleaner.

Reply:
We added the default message in the StepAction component instead of the Loader itself because in the other way we would lose the ability to show the loader animation by itself, without any text. 
We could add another prop to the Loader component to handle the case, like an optional boolean to decide to not show the default message, but that would require a compatibility check over the whole portal to ensure we are not modifying any existing behaviour. It looked more practical to only add it to this component given the situation, but we can do it if you think it's going to be better

Reply:
Did not think of the display w/o message. Good catch! I do think however, that this is a functionality that is directly related to and should be provided by the `Loader` component itself. Have a boolean `showMessage` that is set to true by default for example.